home/import_content_pages.py

```diff
@@ -44,6 +44,8 @@ def __init__(
         progress_queue: Queue[int],
         purge: bool | str = True,
         locale: Locale | str | None = None,
+        import_warnings: ImportWarning | list | None = None
+        # import_warnings: list[ImportWarning] | list | None = None
```

Comment:
No need to have `import_warnings` as a parameter here -- we always want it to start as an empty list.